open-craft/opencraft
# SE-224 Move to using django.contrib.postgres.fields.JSONField

TASK: [SE-224](https://tasks.opencraft.com/browse/SE-224)

This PR explicitly uses `django.contrib.postgres.fields` module for
the `JSONField` model's fields class, replacing the `django_extensions`
solution and definitions.

It runs a migration in order to alter the column type from `TEXT` to
Postgresql's `JSONB` (which is used by the `JSONField` class from the
`django.contrib.postgres.fields` module).

### Test Instructions
- First, you need to apply the last migration from this PR
- Run the project's tests
- Make sure it doesn't trigger any error

### Reviewers
